api_client/python/timesketch_api_client/sketch.py

```diff
@@ -185,14 +211,38 @@ def get_aggregation(self, aggregation_id):
                 return aggregation_obj
         return None
 
+    def get_story(self, story_id=None, story_title=None):
+        """Returns a story object that is stored in the sketch.
+
+        Args:
+            story_id: an integer indicating the ID of the story to
+                be fetched. Defaults to None.
```

Comment:
Isn't this argument required?

Reply:
no, either one... if need to provide either one, if you provide story_title the first instance of a story with that title will be returned... made that a bit more clear in the documentation.

Reply:
hmm, is this really what we want? Oh, a Story for that sketch right? that would make sense

Reply:
Ok, this is for the sketch. All good